wgpu-core/src/track/mod.rs

```diff
@@ -480,6 +480,22 @@ impl<I: Copy + fmt::Debug + TypedId> ResourceState for PhantomData<I> {
 
 pub const DUMMY_SELECTOR: () = ();
 
+#[derive(Clone, Debug, Error)]
+pub enum UsageConflict {
+    #[error("buffer {id:?} combined use is {combined_use:?}")]
```

Comment:
Could these errors provide a bit more description of what they are/what the user has to do to fix them? If I saw this error with no other context I would be very confused and have to go digging.

Reply:
reworded this a bit... still could be improved

Reply:
That's significantly better, I think the only question I have would be "what is a usage scope?", as that terminology isn't on the api surface at all.

Reply:
Usage scopes are defined in upstream spec. We could also have something in wgpu or wgpu-rs docs. Not sure how to link to that from the error message though.

Reply:
Fair enough, this is a doc issue so out of scope.